### PR TITLE
clang-tidy modernize-use-default-member-init and modernize-use-emplace

### PR DESCRIPTION
**Pull request by greglandrum**
Two more clang-tidy modernizations.

### CODE REVIEW

File: Code/Geometry/point.h [hunk 1]
```diff
@@ -45,9 +45,9 @@ class RDKIT_RDGEOMETRYLIB_EXPORT Point {
 // typedef class Point3D Point;
 class RDKIT_RDGEOMETRYLIB_EXPORT Point3D : public Point {
  public:
-  double x, y, z;
+  double x{0.0}, y{0.0}, z{0.0};
 
-  Point3D() : x(0.0), y(0.0), z(0.0){};
```
Comment (bp-kelley) [inline Code/Geometry/point.h:50]: Swig isn't liking this for the java builds:

```
2020-05-26T06:13:34.7240693Z /home/vsts/work/1/s/Code/Geometry/point.h:48: Error: Syntax error in input(3).
```

I think we may need. swig update.

**Comment by bp-kelley** [comment]
@greglandrum I tried out the 'resolve conflicts' and apparently I can push to your repo which I find amusing.  Could you take a look when you get a chance?

**Comment by greglandrum** [comment]
> @greglandrum I tried out the 'resolve conflicts' and apparently I can push to your repo which I find amusing. Could you take a look when you get a chance?

yeah, since you're a maintainer you can generally commit to forks when people do a PR (there's a checkbox they can do when creating the PR to avoid that).

**Comment by greglandrum** [comment]
that merge looks fine to me.

**Pull request merged**